e2e-tests/src/tests/xmodel-app.spec.ts

```diff
@@ -0,0 +1,11 @@
+/********************************************************************************
```

Comment:
Just out of curiosity: I see that you use `xmodel` instead of `crossmodel`, is that something we should apply to more files?

Reply:
That was an oopsy :). I renamed the files.